deps/build.jl

```diff
@@ -121,7 +122,17 @@ end
 # need to be able to get the version before Python is initialized
 Py_GetVersion(libpy) = bytestring(ccall(Libdl.dlsym(libpy, :Py_GetVersion), Ptr{UInt8}, ()))
 
-const python = get(ENV, "PYTHON", "python")
+use_conda = false
+const python = try 
+    pyconfigvar(get(ENV, "PYTHON", "python"),"VERSION","")
+    get(ENV, "PYTHON", "python")
+catch e1
+    info( "No system-wide Python was found; got the following error:\n",
+    "$e1\nusing the Python distribution in the Conda package")
+    use_conda = true
+    abspath(Conda.PREFIX, "python" * (@windows? ".exe" : ""))
```

Comment:
I am tired of windows/linux differences. On linux it should be `abspath(Conda.BINDIR, "python"))` on windows it should be `abspath(Conda.PREFIX, "python".exe"))`. This almost dictates another constant in Conda called `PYTHONDIR` else we should adjust PyCall in this special case.